app/backend/src/tests/test_fixtures.py

```diff
@@ -31,15 +31,24 @@
 )
 
 
-@pytest.fixture
-def db():
+@pytest.fixture(params=[False, True])
+def db(request):
```

Comment:
Ooh, that was super simple :)
The only thing to complain about is the test summary has some nondescriptive True and False strings:
```
ERROR test_api.py::test_ping[False] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could no...
ERROR test_api.py::test_ping[True] - sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not...
```
It will be slightly better output if you had `params=["migrations", "model"]` maybe.

Reply:
> It will be slightly better output if you had `params=["migrations", "model"]` maybe.

I agree. I'll change this.